source/main.lua

```diff
@@ -24,12 +24,15 @@ local groundSprite <const> = gfx.sprite.new(groundImage)
 
 -- create ground obstacles player jumps over
 local groundObstacleImage <const> = gfx.image.new(20, 30, gfx.kColorBlack)
-local groundObstacleCount <const> = 5 -- amount of obstacles in each "round"
-local baseGroundObstacleSpeed <const> = -2 -- obstacles are moving left, so this is negative
+local groundObstacleImage2 <const> = gfx.image.new(20, 80, gfx.kColorBlack)
+local groundObstacleCount <const> = 5  -- amount of obstacles in each "round"
+local baseGroundObstacleSpeed <const> = -2  -- obstacles are moving left, so this is negative
+
 local maxGroundObstacleSpeed <const> = -5
 local groundObstacles = {} -- holds all obstacles, we do this because we need to keep track of multiple obstacles
 local groundObstacleXValues = {}
 local addedGroundObstacleSpeed = 0
+local tallObstacle = false
```

Comment:
this as well; it's not immediately obvious that this is a bool, and i think something like `tallObstacleAppeared` better represents how this is used